libraries/chain/include/eosio/chain/webassembly/eos-vm-oc/config.hpp

```diff
@@ -20,7 +24,11 @@ struct config {
    // libtester disables the limits in all tests, except enforces the limits
    // in the tests in unittests/eosvmoc_limits_tests.cpp.
    std::optional<rlim_t>   cpu_limit {20u};
+#if __has_feature(undefined_behavior_sanitizer) || __has_feature(address_sanitizer)
+   std::optional<rlim_t>   vm_limit; // UBSAN & ASAN can add massive virtual memory usage; don't enforce vm limits when either of them are enabled
```

Comment:
Don't need this for tests to pass: OC in unit tests always has `vm_limit` disabled now (except for the one modified test case), and python integration tests never use OC. But I'd like to use OC + ASAN/UBSAN with nodeos. Such usage is probably "developer enough" where simply disabling this limit is safe when configured that way; i.e. won't trap unsuspecting users. Alternatively could guard this disablement behind `EOSVMOC_ENABLE_DEVELOPER_OPTIONS` but ultimately I want to remove that option.